src/Storefront/Framework/Cookie/GooglePayCookieProvider.php

```diff
@@ -15,13 +15,18 @@
 use Shopware\Storefront\Framework\Cookie\CookieProviderInterface;
 use Swag\PayPal\Checkout\Payment\Method\GooglePayHandler;
 
+/**
+ * @deprecated tag:v11.0.0 - Will be removed in 11.0.0. Use {@see CookieGroupCollectEvent} instead to introduce cookies.
```

Comment:
As the tag already states the version:
```suggestion
 * @deprecated tag:v11.0.0 - Will be removed. Use {@see CookieGroupCollectEvent} instead to introduce cookies.
```